.circleci/config.yml

```diff
@@ -295,6 +299,8 @@ jobs:
       - utils/checkout-with-mise
       - run:
           name: forge test
+          environment:
+            ETH_RPC_URL: << pipeline.parameters.l1_mainnet_rpc_url >>
```

Comment:
Same, we have forge tests that fork from both mainnet and sepolia

Reply:
Can you point me to the place where we fork from sepolia?

Reply:
https://github.com/ethereum-optimism/superchain-ops/blob/877329654a1e3d731b58524ddd7a0f1eedf2abe1/test/tasks/NestedMultisigTask.t.sol#L223